### PR TITLE
Hide the `reclaim-mannequin --skip-invitation` option while it's under development

### PR DESCRIPTION
**Pull request by timrogers**
This hides the `reclaim-mannequin` subcommand's `--skip-invitation` option which is currently in development and not available to customers.

- [ ] Did you write/update appropriate tests
- [ ] Release notes updated (if appropriate)
- [ ] Appropriate logging output
- [ ] Issue linked
- [ ] Docs updated (or issue created)
- [ ] New package licenses are added to `ThirdPartyNotices.txt` (if applicable)

### CODE REVIEW

**Comment by timrogers** [comment]
I'm not going to add this to the release notes as the actual option has not yet been released, and this is a minor change to its documentation.

**Comment by dylan-smith** [comment]
Would it be better to just make it a hidden option while flagged?

**Comment by timrogers** [comment]
I'm going to keep the change to the description, but add `IsHidden` too. See 7d1d974.

**Review by hfishback01 — DISMISSED** [review]
Thanks for doing this, Tim! Looks like tests are failing but the changes look good otherwise. Can approve on passing tests :)

File: src/OctoshiftCLI.Tests/bbs2gh/Commands/ReclaimMannequin/ReclaimMannequinCommandTests.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public void Should_Have_Options()
         TestHelpers.VerifyCommandOption(command.Options, "target-user", false);
         TestHelpers.VerifyCommandOption(command.Options, "force", false);
         TestHelpers.VerifyCommandOption(command.Options, "github-pat", false);
-        TestHelpers.VerifyCommandOption(command.Options, "skip-invitation", false);
+        TestHelpers.VerifyCommandOption(command.Options, "skip-invitation", true);
```
Comment (dylan-smith) [inline src/OctoshiftCLI.Tests/bbs2gh/Commands/ReclaimMannequin/ReclaimMannequinCommandTests.cs:23]:
```suggestion
        TestHelpers.VerifyCommandOption(command.Options, "skip-invitation", false, true);
```

File: src/OctoshiftCLI.Tests/gei/Commands/ReclaimMannequin/ReclaimMannequinCommandTests.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@ public void Should_Have_Options()
         TestHelpers.VerifyCommandOption(command.Options, "target-user", false);
         TestHelpers.VerifyCommandOption(command.Options, "force", false);
         TestHelpers.VerifyCommandOption(command.Options, "github-target-pat", false);
-        TestHelpers.VerifyCommandOption(command.Options, "skip-invitation", false);
+        TestHelpers.VerifyCommandOption(command.Options, "skip-invitation", true);
```
Comment (dylan-smith) [inline src/OctoshiftCLI.Tests/gei/Commands/ReclaimMannequin/ReclaimMannequinCommandTests.cs:23]:
```suggestion
        TestHelpers.VerifyCommandOption(command.Options, "skip-invitation", false, true);
```

**Review by brianaj — APPROVED** [review]
👍🏾

**Review by synthead — APPROVED** [review]

**Pull request merged**